src/pdsfile/pdsfile.py

```
@@ -3017,7 +3021,7 @@ def infoshelf_path_and_key(self):
             try:
                 self._infoshelf_path_and_key = \
                     cls.shelf_path_and_key_for_abspath(self.abspath, 'info')
-            except:
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)